public/index.html

```diff
@@ -30,8 +30,9 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
-    <script crossorigin="anonymous" src="https://clinicaltables.nlm.nih.gov/lforms-versions/25.1.2/lforms.min.js"></script>
-    <script crossorigin="anonymous" src="https://clinicaltables.nlm.nih.gov/lforms-versions/25.1.2/fhir/lformsFHIRAll.min.js"></script>   
+
+    <script crossorigin="anonymous" src="./static/dist/lforms-26.2.0/lforms.min.js"></script>
+    <script crossorigin="anonymous" src="./static/dist/lforms-26.2.0/fhir/lformsFHIRAll.min.js"></script>
```

Comment:
```suggestion
    <script crossorigin="anonymous" src="./static/lforms/lforms-26.2.0/lforms.min.js"></script>
    <script crossorigin="anonymous" src="./static/lforms/lforms-26.2.0/fhir/lformsFHIRAll.min.js"></script>
```
Same change as above